lib/index.js

```diff
@@ -480,6 +480,13 @@ InputMask.prototype.getValue = function getValue() {
   return this.value.join('')
 }
 
+InputMask.prototype.hasValueChanged = function hasValueChanged(nextValue) {
+  if (!nextValue) {
+    nextValue = ''
+  }
+  return this.value.join('') !== this.pattern.formatValue(nextValue.split('')).join('')
```

Comment:
Why is this doing a `join`? (Apologies if the question is a bit naive. I have access to this repo, but haven’t read any of the implementation)